sfdoc/publish/migrations/0037_add_whitelist_model.py

```diff
@@ -0,0 +1,35 @@
+# Generated by Django 2.2.3 on 2019-12-06 01:05
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+def ImportEnvVar(apps, schema_editor):
+    WhitelistedLinkset = apps.get_model('publish', 'WhitelistedLinkset')
+    old_whitelist = getattr(settings, "WHITELIST_URL", None)
+    if old_whitelist:
+        env_vars = "\n".join(old_whitelist)   
+        WhitelistedLinkset.objects.create(name="IMPORTED URLS", urls=env_vars)
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('publish', '0036_merge_20190723_1853'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='WhitelistedLinkset',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('urls', models.TextField()),
+                ('name', models.CharField(max_length=100, null=True, unique=True)),
+            ],
+        ),
+        migrations.AlterField(
+            model_name='bundle',
+            name='status',
+            field=models.CharField(choices=[('N', 'New'), ('Q', 'Queued'), ('C', 'Processing'), ('D', 'Ready for Review'), ('R', 'Rejected'), ('G', 'Publishing'), ('W', 'Waiting to Publish'), ('P', 'Published'), ('E', 'Error')], default='N', max_length=1),
+        ),
```

Comment:
Is this a missing migration for a model change that already happened, or what?

Reply:
Yes. It's one of those migrations that doesn't actually change the schema, so I didn't know I should make a migration.

Reply:
We should make the tests fail if there are migrations which need to be generated.